corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -228,7 +228,7 @@
     <body>
         {% maintenance_alert %}
         <!-- .hide used for A/B test of account creation page -->
-        <div class="hq-container{% if is_register_user_experiment %} hide{% endif %}">
+        <div class="hq-container{% if is_register_user %} molly{% endif %}">
```

Comment:
is `is_register_user` ever `false`? if not this could be simplified. also nit picky but could potentially use a more descriptive (in general ways) class name.